src/Compilers/CSharp/Test/Semantic/IOperation/IOperationTests_ArrayCreationAndInitializer.cs

```diff
@@ -0,0 +1,415 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+using Roslyn.Test.Utilities;
+using Xunit;
+
+namespace Microsoft.CodeAnalysis.CSharp.UnitTests
+{
+    public partial class IOperationTests : SemanticModelTestBase
+    {
+        [Fact, WorkItem(17596, "https://github.com/dotnet/roslyn/issues/17596")]
+        public void SimpleArrayCreation_PrimitiveType()
+        {
+            string source = @"
+class C
+{
+    public void F()
+    {
+        var a = /*<bind>*/new string[1]/*</bind>*/;
+    }
+}
+";
+            string expectedOperationTree = @"
+IArrayCreationExpression (Dimension sizes: 1, Element Type: System.String) (OperationKind.ArrayCreationExpression, Type: System.String[])
```

Comment:
>System.String [](start = 60, length = 13)

A question for design team, is there a value in duplicating the element type in a special property, can't we get it from the type for the node?

Reply:
This PR went dormant and I forgot to file an issue for this. Opened https://github.com/dotnet/roslyn/issues/22265